data_demos/NF_hackathon_exomeseq_datademo.Rmd

```diff
@@ -0,0 +1,216 @@
+---
+title: "Genomic Variants: Introduction to exomeSeq data"
+author: Jineta Banerjee
+date: '`r format(Sys.time(), "%d %B, %Y")`'
+output: 
+  html_document:
+    code_folding: hide
+    toc: true
+    number_sections: true
+    toc_float:
+      collapsed: false
+      smooth_scroll: false
+    fig_width: 7
+    fig_height: 6
+    fig_caption: true
+    df_print: paged 
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+# Getting Started
+
+This document aims to guide challenge particpants who have limited exposure to Synapse as a platform and clinical/biological data, through exploration and analysis of the available exomeSeq datasets. The available dataset has been carefully curated to include genomic variant (exomeSeq) data from normal and four different tumor types from four different study initiatives into a single platform (a harmonized dataset). 
+
+The scope of this document is to inform participants:
+
+* how to navigate the Synapse environment to access and work with the data files
+* display the key features of the genomic variant (exomeSeq) dataset 
+* introduce few general analyses steps of the data and their interpretation
+
+At the end of this document, the participants will be able to download the data and understand various intricacies that exist in this given dataset. We encourage participants to visualize the dataset by faceting on different variables to get an accurate understanding of the dataset. 
+
+## Loading Analytical Packages
+
+Lets first load the R packages we need to query, download, and analyze the data that is stored on Synapse.
+```{r lib_synapser, eval=TRUE, results='show', message=FALSE, warning=FALSE, include=TRUE}
+
+library(synapser)
+library(synapserutils)
+library(BiocManager)
+library(gProfileR)
+library(tidyverse)
+library(DT)
+library(maftools)
+library(here)
+library(ggfortify)
+library(stringr)
+library(reshape2)
+
+```
+
+## Logging in to Synapse
+
+Please download your R Synapse client: [R client](https://r-docs.synapse.org/) and refer to the necessary docs [synapser docs](https://r-docs.synapse.org/articles/synapser.html#connecting-to-synapse) to successfully log in to synapse and start using its various analytical tools.
+
+```{r login_to_synapse, eval=TRUE, results='hide', message=FALSE, warning=FALSE}
+
+## The login credentials have been stored in .synapseConfig file in the home dir. If you dont have this setup, please make the file before trying to login.
+synLogin()
+
+```
+
+## Accessing or downloading data from Synapse
+
+You can look up the SynID of the required file and download it using the __synGet__ command of SynapseR. Load the required csv file by adding __$path__ to the accessed file.
```

Comment:
library is just called `synapser` - no caps. 

Can you add a one sentence description/reminder of what data this synapse entity contains?